docs/application/native/api/iot-headed/4.0/index.html

```diff
@@ -106,7 +106,7 @@ <h2><a class="anchor" id="NATIVE_API_REFERENCE_SECTION1"></a>
  </p>
 <h2><a class="anchor" id="NATIVE_API_REFERENCE_SECTION2"></a>
 Related Feature</h2>
-<p>Some of the Tizen native APIs require features to prevent your application from being shown in the application list on the Tizen store. If related Feature is included in API reference as shown below and your application uses that feature, then you need to declare the feature in the tizen-manifest.xml file. For more information, see <a href="https://developer.tizen.org/development/getting-started/native-application/understanding-tizen-programming/application-filtering"><b>Application Filtering</b>.</a> </p>
+<p>Some of the Tizen native APIs require features to prevent your application from being shown in the application list on the Tizen store. If related Feature is included in API reference as shown below and your application uses that feature, then you need to declare the feature in the tizen-manifest.xml file. For more information, see <a href="https://docs.tizen.org/application/native/tutorials/details/app-filtering"><b>Application Filtering</b>.</a> </p>
```

Comment:
Just a question.

IoT Headed / Headless API are based on Tizen Native API. Is it the reason of keep using the "Tizen native API" in the pages?



![image](https://user-images.githubusercontent.com/9958374/100057205-53118400-2e6a-11eb-8665-c07da5ead7f9.png)

Reply:
Iot Profiles support Native API. but, different coverage compared with Mobile / Wearable.